lnwire/extra_bytes.go

```diff
@@ -269,3 +269,41 @@ func MergeAndEncode(knownRecords []tlv.RecordProducer,
 
 	return EncodeRecords(sortedRecords)
 }
+
+// ParseAndExtractExtraData parses the given extra data into the passed-in
+// records, then returns any remaining records as extra data.
+func ParseAndExtractExtraData(allTlvData ExtraOpaqueData,
```

Comment:
Can we just use `ParseAndExtractCustomRecords` here?

Reply:
We could - but I wanna make it clear that here we don't have custom records.